wallet-webapps-common/src/main/webapp/WEB-INF/conf/wallet/dynamic-container-configuration.xml

```diff
@@ -66,6 +66,53 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         </object-param>
       </init-params>
     </component-plugin>
+    <component-plugin>
+      <name>addPlugin</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.commons.addons.AddOnPluginImpl</type>
+      <description></description>
+      <init-params>
+        <value-param>
+          <name>priority</name>
+          <value>5</value>
+        </value-param>
+        <value-param>
+          <name>containerName</name>
+          <value>bottom-all-container</value>
```

Comment:
Let's add it just for conainer in perkstore page **perk-store-container** or **bottom-perk-store-container**